docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
+    "label": "",
+    "profile": ""
+  },
+  {
+    "name": "application-c",
+    "label": "",
+    "profile": ""
+  },
+  {
+    "name": "atael",
+    "label": "",
+    "profile": ""
+  }
+]
+```
+
+#### /srv/config/all?service-profile=\<profile-name\>
+
+Get all distinct services filtered on profile (service-profile).
+
+```shell
+curl -s http://localhost:8080/srv/config/all? \
+  service-profile=dev
+```
+
+Example of data returned:
+
+```json
+[
+  {
+    "name": "atael",
+    "label": "latest",
+    "profile": "dev"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>
+
+Get all properties for a service-name (application).
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-a
+```
+
+Example of data returned:
+
+```json
+[
+  {
+    "id": 3,
+    "application": "application-a",
+    "profile": "production",
+    "label": "12c",
+    "propKey": "db-name",
+    "value": "databasename-a-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 4,
+    "application": "application-a",
+    "profile": "production",
+    "label": "12c",
+    "propKey": "db-connection",
+    "value": "connectionstring-a-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 5,
+    "application": "application-a",
+    "profile": "development",
+    "label": "23cbeta",
+    "propKey": "db-dev-name",
+    "value": "databasename-a-dev",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 6,
+    "application": "application-a",
+    "profile": "development",
+    "label": "23cbeta",
+    "propKey": "db-dev-connection",
+    "value": "connectionstring-a-dev",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>
+
+Get all properties for a service-name (application) filtered on service-label (label).
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-b& \
+  service-label=19c
+```
+
+Example of data returned:
+
+```json
+[
+  {
+    "id": 7,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-name",
+    "value": "databasename-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 8,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-connection",
+    "value": "connectionstring-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>&service-profile=\<service-profile\>
+
+Get all properties for a service-name (application) filtered on service-profile (profile) and service-profile (profile).
```

Comment:
Change the period to a colon.

You have used "service-profile (profile)" twice in this sentence. Is that intended?